### ISSUE TITLE
Make menril more like vanilla wood

### ISSUE THREAD

**Issue by josephcsible** [CyclopsMC/IntegratedDynamics#272]
I propose the following changes/additions to make menril more like vanilla wood:

- [x] Rename "Menril Log" to "Menril Wood"
- [x] Rename "Enriched Menril Log" to "Enriched Menril Wood"
- [x] Rename "Menril Plank" to "Menril Wood Planks"
- [x] Rename "Menril Plank Stairs" to "Menril Wood Stairs"
- [ ] Add a "Menril Wood Slab"
- [ ] Add a "Menril Fence"
- [ ] Add a "Menril Fence Gate"
- [x] Add a "Menril Door"
- [ ] ~~Add a "Menril Boat"~~
- [x] Make menril planks usable in vanilla crafting recipes where any vanilla wood works (e.g., sticks) (already done in new Forge versions)
- [ ] Make menril planks usable to repair wooden tools in an anvil (blocked by MinecraftForge/MinecraftForge#4397)
- [x] Make menril logs smelt to charcoal

Do these changes sound okay? If so, I'll start working on a PR for them.

----

EDIT by @rubensworks: Do this stuff also for Crystalized Menril Blocks

**Comment by rubensworks** [comment]
Sounds good!

Aren't the following things possible already?
> * Make menril planks usable in vanilla crafting recipes where any vanilla wood works (e.g., sticks)
> * Make menril planks usable to repair wooden tools in an anvil
> * Make menril logs smelt to charcoal

**Comment by josephcsible** [comment]
If they are possible, JEI isn't showing them as such. I'll double-check, though.

**Comment by josephcsible** [comment]
Hmm, it looks like all of your blocks are `IConfigurableBlock`s. Does this mean I should add `ConfigurableBlockSlab`, `ConfigurableBlockFence`, etc. to CyclopsCore, then subclass those to make these new items?

**Comment by josephcsible** [comment]
> - Make menril planks usable in vanilla crafting recipes where any vanilla wood works (e.g., sticks)

Upon re-testing, this does appear to already work. I think the problem may have been I was on an old version of Forge.

> - Make menril planks usable to repair wooden tools in an anvil
> - Make menril logs smelt to charcoal

However, these things don't. This is because neither the furnace nor the anvil support oredict recipes. I'll add these possibilities by hand. (Charcoal depends on CyclopsMC/CyclopsCore#71 since that's supposed to give 0.15 XP.)

**Comment by ghost** [comment]
If I could make a suggestion, Menril chests, like Quark's chests for the vanilla woods.
(Unlimited Chisel Works has menril integration for Chisel.)

**Comment by Tarod7** [comment]
Man I'd love some Menril chests. Great art style in this mod.

**Comment by ghost** [comment]
Charset Storage has a chests mod that does menril chests, but IIRC they don't necessarily play well with other mods.

**Comment by lynaevm** [comment]
Seconding this... specifically for the slab, which seems a minor enough feature.  Beautiful building material, really missing the slab.
